docs/meshstack.azure.index.md

```diff
@@ -395,3 +402,112 @@ The arguments available here are:
 3. argument: meshProject [ID (numeric)](./meshstack.configuration.md#identifiers)
 4. argument: tenantNumber (numeric), a running number specific to each platform which can optionally be enabled
 5. argument: role name suffix ([configurable via Landing Zone](./meshstack.azure.landing-zones.md#meshrole-to-platform-role-mapping))
+
+## Permission Model
+<!-- markdownlint-disable MD001 -->
+
+meshstack in total needs up to three service principals, one for replicating meshProjects into the Azure platform, one if AAD should be used as an user lookup when users are added to a customer or project inside the meshfed panel and one for [gathering metering data](./meshstack.azure.metering.md) for the billing subsystem.
+
+#### meshReplicator Project Provisioning
+
+- `Directory.Read.All` - this permission is required to search the directory for existing users, groups and service principals
+- `Group.ReadWrite.All` - this permissions is required to create new groups
+- `User.Invite.All` - this permission is required if you want to enable B2B User Invitations
+- `User Access Administrator` - to add and remove user group access to the Subscriptions
+- `Rights to call Azure Functions` - see the [end of this section](#azure-function-access-setup) for how to setup this
+
+Created Subscriptions will have the Service Principal of the replicator registered as owner at first. As soon as all needed maintainance steps are performed (like renaming the subscription, moving it to the final management group etc.), the replicator removes itself as an owner.
+
+All permissions left are therefore granted only via the management group hierarchy. The meshstack software does **not** need access related to actual workload inside this subscriptions, however to perform certain maintenance tasks the following permissions/roles must be granted to the replicator principal through the management group hierarchy:
+
+- `Blueprint Operator` - required for assigning and managing Blueprint assignments to the Subscriptions
+- `Management Group Contributor` - required on the target management group for moving Subscriptions
+
+In order to prevent that the replicator is able to assign himself more permissions than originally anticipated its recommended to add a policy rule like this:
```

Comment:
```suggestion
In order to prevent that the replicator from assigning itself more permissions than originally anticipated, it is recommended to add the following policy rule:
```